server/index.ts

```diff
@@ -0,0 +1,4 @@
+import app from './app';
+
+const port = app.get('port');
+app.listen(port, () => console.log(`Running on http://localhost:${port}`));
```

Comment:
I'm surprised that eslint says nothing about this console log, which tell me that maybe eslint doesn't work at the server?